modules/nw-ingress-setting-select-subnet-LoadBalancerService.adoc

```diff
@@ -0,0 +1,89 @@
+// Modules included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-select-subnet-LoadBalancerService_{context}"]
+= Choosing subnets while creating a LoadBalancerService ingress controller
+Users may have dedicated subnets for their load balancers due to security reasons or infrastructure constraints. A cluster administrator can manually set the subnets when creating an ingress controller of type LoadBalancer Service.
+
+Previously, when creating an Ingress Controller, AWS automatically selected the subnets. However, the AWS LoadBalancer Service allows specifying the target subnet using the `service.beta.kubernetes.io/aws-load-balancer-subnets` annotation.
+
+[NOTE]
+====
+This feature and procedure applies only to AWS. 
+====
+
+.Prerequisites
+* The following assumes that you already created an Ingress Controller
+
+.Procedure
+To create an Ingress Controller of type **LoadBalancerService** in Red Hat OpenShift Container Platform 4.17, you can follow these steps:
+
+* Create a custom resource (CR) file.
+
+** Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+    kind: IngressController
+    metadata:
+      namespace: openshift-ingress-operator
+      name: <name>
+    spec:
+      domain: <domain>
+      endpointPublishingStrategy:
+        type: LoadBalancerService
+        loadBalancer:
+          scope: External
+      dnsManagementPolicy: Managed
+----
++
+Replace `<name>` with a name for the Ingress Controller and `<domain>` with the DNS name serviced by the Ingress Controller.
++
+** Apply the CR file:
+
+Save the file and apply it using the OpenShift CLI:
++
+[source,terminal]
+----
+$  oc apply -f sample-ingress.yaml
```

Comment:
Since providing subnets can have mistakes, I'd recommend adding a step here to check the IngressController status conditions for issues:
```
** Confirm the load balancer was provisioned successfully by checking the IngressController conditions. 

    oc get ingresscontroller -n openshift-ingress-operator <name> -o jsonpath="{.status.conditions}" | yq -PC
```